view/Response.class.php

```diff
@@ -352,6 +353,14 @@ public static function getStatusTextForCode($code)
     return $statusTexts[$code] ?? null;
   }
 
+  public static function setPixelAnalyticsType($type){
```

Comment:
Please put the word "Facebook" both in the protected variable name and function name.

Alternatively, this functionality could be dropped, since I think we're going to fire `PageView` on every page.